elpis/wrappers/objects/model.py

```diff
@@ -167,7 +144,7 @@ def prepare_for_training():
 
             # task make-nonsil-phones > {{ .KALDI_OUTPUT_PATH }}/tmp/nonsilence_phones.txt
             nonsilence_phones_path = kaldi_data_local_dict.joinpath('nonsilence_phones.txt')
-            cmd = f"grep -v '^#' < {self.l2s_path} | cut -d' ' -f2 | grep -v '^$' | sort -u"
+            cmd = f"grep -v '^#' < {self.pron_dict.l2s_path} | cut -d' ' -f2 | grep -v '^$' | sort -u"
```

Comment:
This is very clever, but as I've learnt the hard way with Kaldi, bash commands are largely incomprehensible once written, so may be worth an explanatory comment if it's too much work to do in python.

Reply:
Good point!